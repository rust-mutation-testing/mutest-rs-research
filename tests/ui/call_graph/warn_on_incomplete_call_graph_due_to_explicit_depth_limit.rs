//@ print-call-graph
//@ print-targets
//@ stdout
//@ stderr
//@ mutest-flags: -v --call-graph-depth-limit=3

fn depth_5() {}

fn depth_4() {
    depth_5();
}

fn depth_3() {
    depth_4();
}

fn depth_2() {
    depth_3();
}

fn depth_1() {
    depth_2();
}

#[test]
fn test() {
    depth_1();
}
