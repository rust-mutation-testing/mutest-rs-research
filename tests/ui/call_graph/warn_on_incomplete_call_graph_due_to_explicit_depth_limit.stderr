warning: incomplete call graph due to explicit depth limit
  |
  = note: call graph depth limit is set to 3
  = note: ignoring 1 caller and its callees

warning: 1 warning emitted

